specs/approved/whole-system-configuration-steps.rst

```diff
@@ -730,16 +727,15 @@ Documentation Impact
 
 * New page created under Administrator's Guide to describe new clean and deploy
   steps
-* Node cleaning documentation [5]_ is updated to describe new clean steps under
+* Node cleaning documentation [6]_ is updated to describe new clean steps under
   Management interface for idrac-redfish
 
 
 References
 ==========
-.. [0] https://docs.openstack.org/ironic/latest/admin/cleaning.html#cleaning-steps
-.. [1] https://docs.openstack.org/ironic/latest/admin/node-deployment.html#node-deployment-deploy-steps
-.. [2] https://docs.openstack.org/ironic/latest/admin/bios.html#apply-bios-configuration
-.. [3] https://opendev.org/openstack/ironic/src/branch/master/ironic/drivers/raid_config_schema.json
-.. [4] https://downloads.dell.com/solutions/dell-management-solution-resources/RESTfulSerConfig-using-iDRAC-REST%20API(DTC%20copy).pdf
-.. [5] https://docs.openstack.org/ironic/latest/admin/cleaning.html
-
+.. [1] https://docs.openstack.org/ironic/latest/admin/cleaning.html#cleaning-steps
+.. [2] https://docs.openstack.org/ironic/latest/admin/node-deployment.html#node-deployment-deploy-steps
+.. [3] https://docs.openstack.org/ironic/latest/admin/bios.html#apply-bios-configuration
+.. [4] https://opendev.org/openstack/ironic/src/branch/master/ironic/drivers/raid_config_schema.json
+.. [5] https://downloads.dell.com/manuals/all-products/esuprt_solutions_int/esuprt_solutions_int_solutions_resources/dell-management-solution-resources_white-papers15_en-us.pdf
```

Comment:
I further updated this link. Note the copyright is 2018, instead of 2017.

Reply:
thanks